packages/paginated-table/README.md

```diff
@@ -172,7 +244,7 @@ Wrapper component around table compnent.
 * Configuration display component based on index
 * CAUTION: If table is in wrapper component provide position of table component
 */
- export const wrapperConfig = [{
+ export const tableLayout = [{
```

Comment:
will renaming wrapperConfig to  tableLayout break any things  if some one upgrade from older version ?

Reply:
reverted